packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -13,6 +13,36 @@ import { viewNames } from '../../comments/CommentsViews';
 
 export const MAX_COLUMN_WIDTH = 720
 
+const POST_DESCRIPTION_EXCLUSIONS: RegExp[] = [/crossposted/i, /epistemic status/i];
```

Comment:
When I ran the test script, I did see one with "(Cross-posted on The Life You Can Save, Intentional Insights, and LessWrong)." go through, so maybe also include `cross-posted`